### PR TITLE
scripts/build Make ShellCheck happy ♥

### PR DESCRIPTION
**Pull request by kalkin**

### CODE REVIEW

File: scripts/build [hunk 1]
```diff
@@ -33,87 +31,87 @@ REQ_PACKAGES="build-pkgs-$COMPONENT.list"
...
     echo "compile any other component (for the same target distribution) first"
 fi
 
-if [ -r $REQ_PACKAGES ] && [ $REQ_PACKAGES -nt chroot-$DIST/home/user/.installed_${COMPONENT}_`basename $REQ_PACKAGES` ]; then
-    sed "s/DIST/$DIST/g" $REQ_PACKAGES > build-pkgs-temp.list
+if [ -r "$REQ_PACKAGES" ] && [ "$REQ_PACKAGES" -nt "chroot-$DIST/home/user/.installed_${COMPONENT}_$(basename "$REQ_PACKAGES")" ]; then
+    sed "s/DIST/$DIST/g" "$REQ_PACKAGES" > build-pkgs-temp.list
     echo "-> Installing $COMPONENT build dependencies in $DIST environment"
-    sudo chroot $PWD/chroot-$DIST yum install -y `cat build-pkgs-temp.list`
+    sudo chroot "$PWD/chroot-$DIST" yum install -y "$(cat build-pkgs-temp.list)"
```
Comment (marmarek) [inline scripts/build]: This is wrong, the package list should be given as separate arguments (so - no ").

Comment (kalkin) [reply scripts/build]: Good catch

Comment (kalkin) [reply scripts/build]: Fixed

**Comment by kalkin** [comment]
I also added a commit, which adds some `# shellcheck disable` directives.

**Pull request merged**